toplevel/coqtop.ml

```diff
@@ -297,7 +300,7 @@ let start_coq custom =
         prerr_endline "See -help for the list of supported options";
         exit 1
       end;
-      init_toploop opts
+      init_toploop opts, opts
```

Comment:
Does this typecheck?

Reply:
I think so (received by the `let state, opts = ...`).